CMake/cdat_modules/r_pkg.cmake

```diff
@@ -1,9 +1,9 @@
 set(R_MAJOR_SRC 3)
-set(R_MINOR_SRC 1)
-set(R_PATCH_SRC 0)
-set(R_URL ${LLNL_URL})
+set(R_MINOR_SRC 2)
+set(R_PATCH_SRC 1)
+set(R_URL "http://cran.r-project.org/src/base/R-3")
```

Comment:
ok I will tweak this back to LLNL when it works, it's "safer" to leave the links pointing to our server, less likely to break in the future.